Rakefile

```diff
@@ -29,6 +29,11 @@ else
   task :build do; end
 end
 
+task :populate_proxy do
+  require 'toxiproxy'
+  Toxiproxy.populate(File.expand_path('../test/fixtures/toxiproxy.json', __FILE__))
+end
```

Comment:
Yeah it's just that there is no test helper yet. I need to fix that. 

Maybe in a followup if you don't mind.